lib/dradis/plugins/acunetix/importer.rb

```diff
@@ -30,7 +30,12 @@ def import(params={})
     attr_accessor :scan_node
 
     def process_scan(xml_scan)
-      start_url = URI::parse(xml_scan.at_xpath('./StartURL').text()).host
+      # Don't URI-parse simple URLs like testphp.vulnweb.com 
+      if xml_scan.at_xpath('./StartURL').text().start_with?('http')
```

Comment:
Nice catch!

But with this change we may be losing some cases, like when the url starts with `https` ?
Instead of going deep with regexes or if cases to catch all protocols, I propose to use something like:
```
url = xml_scan.at_xpath('./StartURL').text()
start_url = URI::parse(url) || url
```

Reply:
@dormi I tried your suggestion and was left with an error: 

```
[21:33:44] There was a fatal error processing your upload:
[21:33:44] can't cast URI::Generic
[21:33:44] /Users/rachkor/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/activerecord-5.1.6.1/lib/active_record/connection_adapters/abstract/quoting.rb:45:in `rescue in type_cast'
```

I tried a few similar logic statements and wasn't able to find one that worked without an error. Any suggestions?

Reply:
I think I missed a `host`:
```
url = xml_scan.at_xpath('./StartURL').text()
start_url = URI::parse(url).host || url
```